### DISCUSSION TITLE
Github fails to send verification SMS, and locks me out of my account

### DISCUSSION THREAD

**Discussion by alphanumeric1d** [community/community#43280] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

correct authentication
correct recovery codes
correct passwords
correct username 
correct email
succesfully received password reset link (refuse to accept new password, because they fail to send a different password reset verification SMS)
Contact support 
wait 3 days (5 days)

**Comment by Niktys** [comment]
``
